src/Date.php

```diff
@@ -243,7 +243,16 @@ public function format($format)
 
                 // Short notations.
                 if (in_array($character, ['D', 'M'])) {
-                    $translated = mb_substr($translated, 0, 3);
+                    $toTranslate     = strtolower($original);
```

Comment:
I'm more than willing to change this to `mb_strtolower`, but the original source code uses `strtolower` as well: https://github.com/jenssegers/date/blob/master/src/Date.php#L241